src/Frame.h

```diff
@@ -149,6 +157,7 @@ class Frame {
 
     // Apply Region/Slicer to image (Frame manages image mutex) and get shape, data, or stats
     casacore::LCRegion* GetImageRegion(int file_id, std::shared_ptr<carta::Region> region);
+    casacore::LCRegion* GetImageRegion(int file_id, std::shared_ptr<carta::Region> region, casacore::IPosition shape);
```

Comment:
This does not appear to be defined or used in `Frame.cc`.

Reply:
Removed it